composer/loggers/progress_bar_logger.py

```diff
@@ -30,17 +42,17 @@ def __init__(
         position: int,
         bar_format: str,
         file: TextIO,
-        metrics: Dict[str, Any],
-        keys_to_log: List[str],
         timestamp_key: str,
+        dataloader_label_to_metrics: Optional[Dict[str, Union[str, bool]]] = None,
+        is_train: bool = False,
```

Comment:
I can change the metrics part. I would like to keep the `is_train` variable since that indicates to log the default `'loss/train' metric. In the previous implementation, there was also a similar variable, it was just a part of the `keys_to_log` field.